src/_pytest/terminal.py

```diff
@@ -1636,3 +1638,14 @@ def _get_raw_skip_reason(report: TestReport) -> str:
         elif reason == "Skipped":
             reason = ""
         return reason
+
+
+class _CallableBool:
```

Comment:
Can move it to `compat.py`? It's sort a dumping ground where we put such things :)

Also no need for the `_` prefix, everything not re-exported through `pytest` (vs `_pytest`) is private.